apis/cloud.redhat.com/v1alpha1/clowdenvironment_types.go

```diff
@@ -239,6 +239,17 @@ type ClowdEnvironmentStatus struct {
 	// Important: Run "make" to regenerate code after modifying this file
 	TargetNamespace string                  `json:"targetNamespace"`
 	Deployments     common.DeploymentStatus `json:"deployments"`
+	Apps            []AppStatus             `json:"apps,omitempty"`
+}
+
+type AppStatus struct {
+	Name        string             `json:"name"`
+	Deployments []DeploymentStatus `json:"deployments"`
+}
+
+type DeploymentStatus struct {
```

Comment:
Some comment here -- `DeploymentStatus` is odd because you're not giving the status.